CRM/Member/Form/MembershipConfig.php

```diff
@@ -83,7 +83,8 @@ public function setDefaultValues() {
     }
 
     if (isset($defaults['minimum_fee'])) {
-      $defaults['minimum_fee'] = CRM_Utils_Money::format($defaults['minimum_fee'], NULL, '%a');
+      // Don't use CRM_Utils_Money::format here as we need to allow more decimal places so sales tax can be calculated with recurring amounts.
+      $defaults['minimum_fee'] = filter_var($defaults['minimum_fee'], FILTER_SANITIZE_NUMBER_FLOAT, FILTER_FLAG_ALLOW_FRACTION);
```

Comment:
Once we've resolved the data format we should train CRM_Utils_Money::format to accept more decimals when appropriate, possibly another parameter to the function, or else wrap it - ie CRM_Utils_Money::formatLongDecimal